File: specificoop.cpp
#include<iostream>
using namespace std;

class Arithmatic
{
    public:
    int iNo1,iNo2;

    Arithmatic(int a,int b)
    {
        iNo1=a;
        iNo2=b;
    }

    int Addition()
    {
        int iAns=0;
        iAns=iNo1+iNo2;
        return iAns;
    }

     int Substraction()
    {
        int iAns=0;
        iAns=iNo1-iNo2;
        return iAns;
    }

};
int main()
{
    int Ret=0;
    Arithmatic obj(10,11);
    Ret=obj.Addition();
    cout<<"Addition is : "<<Ret<<"\n";

    Ret=obj.Substraction();
    cout<<"Substraction is : "<<Ret<<"\n";

    return 0;
}

File: SpecificOOP.java
import java.lang.*;

class Arithmatic
{
    public int No1;
    public int No2;

    Arithmatic(int a,int b)
    {
        No1=a;
        No2=b;
    }
    public int Addition()
    {
        int Ans=0;
        Ans=No1+No2;
        return Ans;
    }
}
class specificOOP
{
    public static void main(String arg[])
    {
        Arithmatic obj=new Arithmatic(10,11);
        int ret=0;
        ret=obj.Addition();
        System.out.println("Addition is : "+ret);
    }
}
